docs/src/main/sphinx/connector/vertica.md

```diff
@@ -0,0 +1,216 @@
+# Vertica connector
+
+```{raw} html
+<img src="../_static/img/vertica.png" class="connector-logo">
+```
+
+The Vertica connector allows querying a [Vertica](https://www.vertica.com/)
+database as an external data source.
+
+## Requirements
+
+To connect to Vertica, you need:
+
+- Vertica 9.1.x or higher.
+- Network access from the coordinator and workers to the Vertica server.
+  Port 5433 is the default port.
+
+## Configuration
+
+Create a catalog properties file in `etc/catalog` named `example.properties` to
+access the configured Vertica database in the `example` catalog. Replace example
+with your database name or some other descriptive name of the catalog. Configure
+the usage of the connector by specifying the name `vertica` and replace the
+connection properties as appropriate for your setup.
+
+```properties
+connector.name=vertica
+connection-url=jdbc:vertica://example.net:5433/test_db
+connection-user=root
+connection-password=secret
+```
+
+The `connection-user` and `connection-password` are typically required and
+determine the user credentials for the connection, often a service user. You can
+use [secrets](/security/secrets) to avoid actual values in the catalog
+properties files.
+
+```{include} jdbc-common-configurations.fragment
+```
```

Comment:
What about these other includes.. I think they also apply maybe.. can you let me know @ebyhr 


```{include} query-comment-format.fragment
```

```{include} jdbc-domain-compaction-threshold.fragment
```

```{include} jdbc-case-insensitive-matching.fragment
```

```{include} non-transactional-insert.fragment
```

Reply:
otherwise @kokosing or @findinpath .. do you know ..

Reply:
{include} non-transactional-insert.fragment does not apply since the connector doesnt support insert

Reply:
I think the others apply so I added them.